src/Hazelcast.Net/Core/InMemoryFormat.cs

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Hazelcast.Serialization;
```

Comment:
Is this needed?

Reply:
Actually, not vital but yes to be more pretty on documentation. The `IData` mention on documentation cannot be resolved otherwise. 

The line https://github.com/hazelcast/hazelcast-csharp-client/blob/a34db92a208304741c7d1b671a19eeece3c64c21/src/Hazelcast.Net/Core/InMemoryFormat.cs#L25

Reply:
ah ok